Gemfile

```diff
@@ -5,7 +5,8 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-
+gem 'figaro'
```

Comment:
i bet we never forget this again

Reply:
For those of us that need to install figaro on this project, to ensure we have all the same keys please see instructions below.
```
... Once merged to master
$ git checkout master
$ git pull origin master
$ bundle
$ bundle exec figaro install
```
From here, open your new `config/application.yml` and add the `GOOGLE_CLIENT_ID:` & `GOOGLE_CLIENT_SECRET:` from our slack channel.